src/main/java/ui/components/pickers/LabelPickerDialog.java

```diff
@@ -208,45 +210,52 @@ private final void populateGroupLabels(List<String> finalLabels, List<String> ma
+                                                  Optional<TurboLabel> suggestion) {
         Map<String, FlowPane> groupContent = new HashMap<>();
         allLabels.stream().sorted()
                 .filter(label -> label.isInGroup())
-                .map(label -> new PickerLabel(label, false))
                 .forEach(label -> {
                     String group = label.getGroupName();
                     if (!groupContent.containsKey(group)) {
                         groupContent.put(group, createGroupPane(GROUP_PAD));
                     }
                     groupContent.get(group).getChildren().add(processMatchedLabel(
-                        label.getFullName(), matchedLabels, finalLabels, suggestion));
+                        label, matchedLabels, finalLabels, suggestion));
                 });
         return groupContent;
     }
 
-    private final void populateGrouplessLabels(List<String> finalLabels, List<String> matchedLabels,
-                                         Optional<String> suggestion) {
+    private final void populateGrouplessLabels(List<TurboLabel> finalLabels, List<TurboLabel> matchedLabels,
+                                         Optional<TurboLabel> suggestion) {
         FlowPane groupless = createGroupPane(GROUPLESS_PAD);
         allLabels.stream()
-                .filter(label -> !label.isInGroup())
-                .map(label -> new PickerLabel(label, false))
-                .forEach(label -> groupless.getChildren().add(processMatchedLabel(
-                    label.getFullName(), matchedLabels, finalLabels, suggestion)));
+            .filter(label -> !label.isInGroup())
+            .forEach(label -> groupless.getChildren().add(processMatchedLabel(
+                label, matchedLabels, finalLabels, suggestion)));
 
         feedbackLabels.getChildren().add(groupless);
     }
 
-    private final Node processMatchedLabel(String repoLabel, List<String> matchedLabels, 
-                                            List<String> assignedLabels, Optional<String> suggestion) {
+    private final Node processMatchedLabel(TurboLabel repoLabel, List<TurboLabel> matchedLabels, 
+                                           List<TurboLabel> assignedLabels, Optional<TurboLabel> suggestion) {
 
         return getPickerLabelNode(
-            new PickerLabel(TurboLabel.getFirstMatchingTurboLabel(allLabels, repoLabel), false)
+            new PickerLabel(TurboLabel.getFirstMatchingTurboLabel(allLabels, repoLabel.getFullName()), false)
                 .faded(!matchedLabels.contains(repoLabel))
                 .highlighted(suggestion.isPresent() && suggestion.get().equals(repoLabel))
                 .selected(assignedLabels.contains(repoLabel)));
     }
 
+    /**
+     * Positions dialog based on width and height of stage
+     * @param stage
+     */
+    private final void positionDialog(Stage stage) {
```

Comment:
To avoid the same mistake again, may be you should document why this method exists and the fact that it affects only some computers.